public_html/wp-content/plugins/wordcamp-reports/views/html/wordcamp-counts.php

```diff
@@ -135,9 +212,14 @@
 				</td>
 
 				<td class="number">
-					<span class="total"><?php echo number_format_i18n( $event['totals']['organizer'] ); ?></span>
+					<span class="total">Total: <?php echo number_format_i18n( $event['totals']['organizer'] ); ?></span>
+					<br>
+					FT: <?php echo number_format_i18n( $event['first_times']['organizer']['yes'] ); ?>
```

Comment:
Is there enough room to write out `First Time Organizers` etc, rather than abbreviating?

Reply:
Yeah, it would be something like:
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/8a719cb7-826f-4fad-aec8-2f2c220c8e80)
I like abbreviations more as there's already full text in the header and the reason I added an extra abbreviated title in every cell is for a long list like below (when scrolling to the bottom, it would be more intuitive if there's a helper text there)
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/b2518bd4-83ae-4d3b-ae91-7e6b5fd9b9b6)

Reply:
Or we can make the header sticky, it doesn't seem not easy to implement it.

https://github.com/WordPress/wordcamp.org/assets/18050944/65577608-e6e0-4b14-b743-8cf4c4658040

Reply:
🤔 Yeah, a sticky header would be nice, but that's probably something that should be done upstream in Core rather than here.

I guess the underlying problem here is that it's so much information to try and squeeze into this format. That's not specific to this PR, though, so it's probably not worth addressing here. I think we can go ahead with what you have, and then revisit the issue later to make the chart easier to digest. That may be something a designer could help with.

Reply:
no problem 👍 
I'll open another issue and ask for feedback from the design and tardis team.

Reply:
Follow-up issue: https://github.com/WordPress/wordcamp.org/issues/993